src/main/java/io/confluent/connect/hdfs/TopicPartitionWriter.java

```diff
@@ -103,22 +108,23 @@ public TopicPartitionWriter(
       HdfsSinkConnectorConfig connectorConfig,
       SinkTaskContext context,
       AvroData avroData) {
-    this(tp, storage, writerProvider, partitioner, connectorConfig, context, avroData, null, null, null, null, null);
+    this(tp, storage, writerProvider, partitioner, connectorConfig, context, avroData, null, null, null, null, null, null);
   }
 
   public TopicPartitionWriter(
-      TopicPartition tp,
-      Storage storage,
-      RecordWriterProvider writerProvider,
-      Partitioner partitioner,
-      HdfsSinkConnectorConfig connectorConfig,
-      SinkTaskContext context,
-      AvroData avroData,
-      HiveMetaStore hiveMetaStore,
-      HiveUtil hive,
-      SchemaFileReader schemaFileReader,
-      ExecutorService executorService,
-      Queue<Future> hiveUpdateFutures) {
+          TopicPartition tp,
```

Comment:
Can you please restore original indentation so changelog will only highlight additions?